man/logind.conf.xml

```diff
@@ -243,14 +245,20 @@
         sleep keys and the lid switch by taking a low-level inhibitor lock
         (<literal>handle-power-key</literal>, <literal>handle-suspend-key</literal>,
         <literal>handle-hibernate-key</literal>, <literal>handle-lid-switch</literal>,
-        <literal>handle-reboot-switch</literal>).
+        <literal>handle-reboot-key</literal>).
         This is most commonly used by graphical desktop environments
         to take over suspend and hibernation handling, and to use their own configuration
         mechanisms. If a low-level inhibitor lock is taken, logind will not take any
         action when that key or switch is triggered and the <varname>Handle*=</varname>
         settings are irrelevant.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>LongpressDuration=</varname></term>
```

Comment:
We generally suffix settings that take time parameters with `Sec=` (even if we allow more fine grained specification, as seconds is our default time unit if people don#t specify anything else).

Also I figure this should be LongPress rather than Longpress.

maybe:

KeyLongPressDurationSec=…

Reply:
Do we really need to make this configurable? 

Taking `CtrlAltDelBurstAction=` as an example, we don't allow to configure the duration but simply document that it is 2s.

Reply:
@robimarko

Reply:
To me, it would be really good to have it configurable.
I don't see a point in hardcoding the time.

Reply:
Every configuration knob comes at a cost. I don't see a compelling reason why not picking a reasonable value like say 4s wouldn't work for everyone.

Reply:
@poettering what's you take on this? Do you think this need to be configurable?

Reply:
i'd probably pick some sane value for now and not make it configurable. And if plenty people then show up and ask for this we can still make it configurable.

(also, we make some settings that are of this "meh, maybe this shouldn't be a primary config knob" configurable via env vars, see doc/ENVIRONMENT.md. If we make this configurable after all one day, we might opt for that.)

Anyway, I think I agree with @mbiebl here, let's just pick some good value here and stick to it for now. (What is the usual long press timeout used by various devices out there? i.e. we should probably follow prior art on the precise time)

Reply:
Ok, if that's the majority opinion.

I have usually seen 5 seconds as a common figure on various boards.

Reply:
Ok, I'l fix it to 5s then, if nobody minds